src/main/java/baseball/model/Computer.java

```diff
@@ -0,0 +1,48 @@
+package baseball.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Computer {
+    public List<Integer> randomNumber;
+
+    public Computer() {
+        saveRandomNumberWithGameStart();
+    }
```

Comment:
save를 통해서 randomNumber에 대한 init을 한다는거는 알겠는데
위의 코드보다는

```suggestion
public Computer() {
    this.randomNumber = generate~~();
}
```

와 같이 수정하는게 더 가독성이 좋을거 같은데 어떻게 생각하시나요?